components/molecule/accordion/src/index.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import Tab from './Tab'
+
+const BASE_CLASS = 'sui-MoleculeAccordion'
+
+class MoleculeAccordion extends Component {
+  state = {
+    openTabs: []
```

Comment:
you can now use React hooks in order to manage state and so you can use function Components instead of a class. https://reactjs.org/docs/hooks-intro.html